pkg/holoscan-networking/CMakeLists.txt

```diff
@@ -16,22 +16,25 @@
 holohub_configure_deb(
   NAME "holoscan-networking"
   DESCRIPTION "Holoscan Networking"
-  VERSION "0.1.0"
+  VERSION "0.0.1"
```

Comment:
Why version downgrade?

Reply:
Our internal formal release process requires an initial archive to bootstrap the infrastructure for releasing it. If I kept 0.1.0, I'd be reserving that name for the test/bootstrap archive, but I'd like to reserve it for the first release.

Reply:
Makes sense, and there has not already been an internal build with 0.1.0?

Reply:
not kicked off no